pkg/cdn/metadata/azurecdn/getter.go

```diff
@@ -0,0 +1,18 @@
+package azurecdn
+
+import (
+	"github.com/gomods/athens/pkg/storage/azurecdn"
+)
+
+// Metadata implements the Getter interface for Azure CDN. It also composes
+// a (github.com/gomods/athens/pkg/storages/azurecdn).Storage implementation
+//
+// for Azure CDN
+type Metadata struct {
+	azurecdn.Storage
+}
+
+// Get implements the Getter interface
+func (m Metadata) Get(module string) (string, error) {
+	return m.BaseURL().String(), nil
```

Comment:
what if CDN has a domain registered, this should be maybe configurable

Reply:
and not tied to storage URL

Reply:
maybe fallback to this

Reply:
I talked to @marstr about this and he had the same point. CDN by default uses account.azureedge.net so that problem is even if there's no domain registered. +1 on making it configurable. Mind if we do that in a follow-up?

Reply:
Ref: https://github.com/gomods/athens/issues/178